MediaBrowser.Controller/Lyrics/LyricResponse.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// LyricResponse model.
+/// </summary>
+public class LyricResponse
+{
+    /// <summary>
+    /// Gets or sets Metadata.
```

Comment:
```suggestion
    /// Metadata for the lyrics
```

Reply:
I can't make this change without the analyzer complaining.

However `Gets or sets Metadata for the lyrics` would work.